internal/web3ext/web3ext.go

```diff
@@ -874,7 +874,13 @@ web3._extend({
 			name: 'getFirstQueueIndexNotInL2Block',
 			call: 'scroll_getFirstQueueIndexNotInL2Block',
 			params: 1
-		})
+		}),
+		new web3._extend.Method({
+			name: 'getBlockByHash',
+			call: 'scroll_getBlockByHash',
```

Comment:
we already have `eth_getBlockByHash` why do we want `scroll_getBlockByHash`?   
or is it a mistake and should be `scroll_getBlockRowConsumptionByHash`